services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -1290,6 +1258,45 @@ public IOFSwitch lookupSwitch(DatapathId dpId) throws SwitchNotFoundException {
                 .collect(Collectors.toList());
     }
 
+    @VisibleForTesting
+    OFInstructionMeter installMeterAndBuildInstruction(IOFSwitch sw, long meterId, long packetsLimit,
+                                                       ArrayList<OFAction> actionList) {
+        long rate = isSwitchSupportsPktpsFlag(sw) ? packetsLimit : getRateForMeterInKbps(packetsLimit);
+
+        OFMeterConfig meterConfig;
+        try {
+            meterConfig = getMeter(sw.getId(), meterId);
+        } catch (SwitchOperationException e) {
+            logger.warn(format("Meter %s won't be installed on the switch %s", meterId, sw.getId()), e);
```

Comment:
Use message provided by exception object. Also we don't need traceback here too.